tests/test_keys/test_umbral_keys.py

```diff
@@ -15,6 +15,34 @@ def test_gen_key():
     umbral_pub_key = umbral_priv_key.get_pubkey()
     assert type(umbral_pub_key) == keys.UmbralPublicKey
 
+def test_derive_key_from_label():
+    
+    umbral_keying_material = keys.UmbralKeyingMaterial()
+
+    label = b"my_healthcare_information"
+
+    priv_key1 = umbral_keying_material.derive_privkey_by_label(label)
```

Comment:
`derive_privkey_by_label` looks a little lengthy, but I cannot think of a better name for the method easily